### PR TITLE
Add Notification-Bar Component 

### LINKED ISSUE #19: Add Notification-Bar Component
As per [Figma](https://www.figma.com/file/qOoG6pKbWKi2ldzHT2cHA6/Crypto-Component---Light-Mode-(Part-2)-(Community)?node-id=1%3A6392&t=Jrp064v41nbf7cAV-4)

### Preview

<div align="center">

Image

</div>

### Variants
- [ ] Light
- [x] Dark

### PR DESCRIPTION
**Pull request by mrxshinji**
Fixed #19

# Added Notification component route:
  ```path: 'components/notification',```

## set font-family, font style and box-sizing @ index.css
 ```
box-sizing: border-box;
font-family: 'Inter';
font-style: normal;
```

image

### CODE REVIEW

File: src/components/notifications/notificationBar.css [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  position: relative;
+
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+  padding: 0px;
+  gap: 15px;
+
+  border-left: 2px solid #f2f2f7;
```
Comment (aftabrehan) [inline src/components/notifications/notificationBar.css:15]: Only have a blank line between selectors.

```suggestion
  position: relative;
  display: flex;
  flex-direction: row;
  align-items: center;
  justify-content: center;
  padding: 0px;
  gap: 15px;
  border-left: 2px solid #f2f2f7;
```

File: src/components/notifications/notificationBar.css [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+.notification-bar-container.active {
+  border-left: 2px solid #5856d6;
+}
+
+.notification-icon {
+  width: 40px;
+  height: 40px;
+
+  background: rgba(88, 86, 214, 0.1);
```
Comment (aftabrehan) [inline src/components/notifications/notificationBar.css:26]:
```suggestion
  height: 40px;
  background: rgba(88, 86, 214, 0.1);
```

File: src/components/notifications/notificationBar.css [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+}
+
+.notification-text {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 0px;
+  gap: 5px;
+
+  width: 246px;
```
Comment (aftabrehan) [inline src/components/notifications/notificationBar.css:44]:
```suggestion
  gap: 5px;
  width: 246px;
```

File: src/components/notifications/notificationBar.css [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+}
+
+.text-title {
+  font-weight: 600;
+  font-size: 13px;
+
+  color: #1c1c1e;
```
Comment (aftabrehan) [inline src/components/notifications/notificationBar.css:51]:
```suggestion
  font-size: 13px;
  color: #1c1c1e;
```

File: src/components/notifications/notificationBar.css [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+.text-title.active {
+  color: #5856d6;
+}
+
+.text-subtitle {
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 15px;
+
+  color: #8E8E93;
```
Comment (aftabrehan) [inline src/components/notifications/notificationBar.css:63]:
```suggestion
  line-height: 15px;
  color: #8E8E93;
```

File: src/components/notifications/notifications.css [hunk 6]
```diff
@@ -0,0 +1,150 @@
+.notification-container {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 30px 20px 20px;
+  gap: 30px;
+
+  position: absolute;
```
Comment (aftabrehan) [inline src/components/notifications/notifications.css:8]:
```suggestion
  gap: 30px;
  position: absolute;
```

File: src/components/notifications/notifications.css [hunk 7]
```diff
@@ -0,0 +1,150 @@
...
+  width: 360px;
+  height: 476px;
+  left: 102px;
+  top: 226px;
+
+  background: #ffffff;
```
Comment (aftabrehan) [inline src/components/notifications/notifications.css:14]:
```suggestion
  top: 226px;
  background: #ffffff;
```

File: src/components/notifications/notifications.css [hunk 8]
```diff
@@ -0,0 +1,150 @@
...
+  justify-content: center;
+  align-items: center;
+
+  width: 19px;
+  height: 19px;
+
+  background: #5856d6;
+  border-radius: 50%;
+
+  font-weight: 600;
```
Comment (aftabrehan) [inline src/components/notifications/notifications.css:62]:
```suggestion
  align-items: center;
  width: 19px;
  height: 19px;
  background: #5856d6;
  border-radius: 50%;
  font-weight: 600;
```

File: src/components/notifications/notifications.css [hunk 9]
```diff
@@ -0,0 +1,150 @@
...
+  flex-direction: column;
+  justify-content: center;
+  align-items: flex-start;
+  padding: 12px;
+  gap: 10px;
+
+  width: 100%;
+  height: 64px;
+
+  background: #5856d6;
```
Comment (aftabrehan) [inline src/components/notifications/notifications.css:107]:
```suggestion
  gap: 10px;
  width: 100%;
  height: 64px;
  background: #5856d6;
```

File: src/components/notifications/notifications.css [hunk 10]
```diff
@@ -0,0 +1,150 @@
...
+}
+
+.notification-sub-text {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 0px;
+  gap: 5px;
+}
+.notification-sub-text span {
```
Comment (aftabrehan) [inline src/components/notifications/notifications.css:136]:
```suggestion
}

.notification-sub-text span {
```

File: src/components/notifications/notifications.css [hunk 11]
```diff
@@ -0,0 +1,150 @@
...
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+
+  color: white;
```
Comment (aftabrehan) [inline src/components/notifications/notifications.css:141]:
```suggestion
  line-height: 16px;
  color: white;
```

File: src/components/notifications/notifications.css [hunk 12]
```diff
@@ -0,0 +1,150 @@
...
+}
+
+.notification-sub-text p {
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 15px;
+
+  color: #d1d1d6;
```
Comment (aftabrehan) [inline src/components/notifications/notifications.css:149]:
```suggestion
  line-height: 15px;
  color: #d1d1d6;
```

File: src/index.js [hunk 13]
```diff
@@ -38,6 +39,10 @@ const router = createBrowserRouter([
         path: 'stories/button',
         element: <ButtonStory />,
       },
+      {
+        path: 'components/notification',
```
Comment (aftabrehan) [inline src/index.js:43]: Just use a single flow to display components.

```suggestion
        path: 'stories/notification',
```

File: src/routes/notification-page/index.jsx [hunk 14]
```diff
@@ -0,0 +1,11 @@
+import Notifications from '../../components/notifications/notifications';
+
+const NotificationPage = () => {
```
Comment (aftabrehan) [inline src/routes/notification-page/index.jsx:3]: `NotificationStory` make sense.

```suggestion
const NotificationStory = () => {
```

File: src/routes/notification-page/index.jsx [hunk 15]
```diff
@@ -0,0 +1,11 @@
...
+  return (
+    <div>
+      <Notifications />
+    </div>
```
Comment (aftabrehan) [inline src/routes/notification-page/index.jsx:7]: Any specific reason for wrapping it in a `div`, as I can see no classes on it?

```suggestion
     <Notifications />
```

**Review by aftabrehan — CHANGES_REQUESTED** [review]
Excellent PR, @mrxshinji 💪 🚀 💯  Again, I'm impressed with your work, I cannot see any bug in the code, which is good. I've requested some code-writing suggestions for you. 

Also try using the short name of a branch `ruc/mrxshinji/add-notification-bar-component`  can be`ruc/mrxshinji/notification-bar`

Thank you 🙏

**Comment by mrxshinji** [comment]
[added hover effect and changes as requested]
(https://github.com/appwex/react-ui-components/pull/26/commits/c737226ce9cc61327d956eda889091e48007d3e1)

**Comment by mrxshinji** [comment]
https://github.com/appwex/react-ui-components/pull/26/commits/2b13492a3aadbb5bb7fecb7455d2f182a0bbc692 
Added link to component

**Review by aftabrehan — APPROVED** [review]
Excellent PR, @mrxshinji 💪 🚀 💯

**Pull request merged**